### ISSUE TITLE
No more compatibility with Grafana 6.3.x?

### ISSUE THREAD

**Issue by StephenSorriaux** [grafana/influxdb-flux-datasource#56]
Hi,

I am running Grafana 6.3.5 and noticed that the the latest (5.4.0) version of the plugin breaks with the following error:

```
my-awesome-flux-dc plugin failed
Error: Fetch error: 404 Not Found Instantiating https://example.com/public/@grafana/slate-react Loading https://example.com/public/plugins/grafana-influxdb-flux-datasource/module.js Loading plugins/grafana-influxdb-flux-datasource/module
```

I did not find any breaking change in the release note, did I miss something?

Thank you for this great plugin, really useful.

**Comment by jimirocks** [comment]
Experiencing exactly the same

**Comment by briangann** [comment]
The plugin is currently only compatible with 6.4.x due to changes in dependencies on the newer grafana/ui components.

We're looking at options for backwards compatibility by including the dependencies in the plugin itself without relying on exposed packages from Grafana.

The current grafana/toolkit build process assumes the target version is 6.4+

**Comment by amotl** [comment]
We just tried it on our local workstation using

- Grafana 6.5.3
- InfluxDB (Flux) Datasource [BETA] 5.4.1

and the plugin **seems to work without any issues**. So, we are wondering if it would be safe to upgrade from Grafana 6.4.5 to the latest Grafana 6.6.0 on our production servers.

**Comment by amotl** [comment]
We just upgraded to Grafana 6.6.0 and the dashboard [Statista: Stockübersicht & Bienenwetter [Testvolk ZK/U]](https://swarm.hiveeyes.org/grafana/d/OvnLNPqiz/statista-stockubersicht-and-bienenwetter-testvolk-zk-u), which uses Flux queries, still works like a charm.

So, we figure `grafana-influxdb-flux-datasource` indeed is compatible with recent versions of Grafana (again).

**Comment by lafrech** [comment]
I just installed Grafana 6.6.2 using official repos

    https://packages.grafana.com/oss/deb stable main

Then I followed instructions in https://grafana.com/grafana/plugins/grafana-influxdb-flux-datasource. I downloaded grafana-influxdb-flux-datasource-5.4.1.tar.gz and copied its content into 

    /usr/share/grafana/public/app/plugins/datasource

(The page linked above says `data/plugins` but I couldn't find that.)

The plugin is listed in the "Others" section of the datasources list but when using it, I get an error:

```
expected expression, got '<' Evaluating http://domain.tld:3000/public/app/plugins/datasource/grafana-influxdb-flux-datasource/module Loading app/plugins/datasource/grafana-influxdb-flux-datasource/module
Data Source Error
```

Is support broken or did I do wrong?

**Comment by StephenSorriaux** [comment]
Hi,

I have a very similar setup (Grafana 6.6.2, this plugin in 5.4.1 but running in Docker) and it works well (query, creation / deletion of datasources, etc.).

**Comment by wetterfrosch** [comment]
> Is support broken or did I do wrong?

@lafrech After updating the datasource-plugin, check its settings: a "default bucket"-setting was introduced and it appears to me that it must be set to some database.

**Comment by lafrech** [comment]
My fault. I tried to follow instructions in README and copied the plugin files into 

    /usr/share/grafana/public/app/plugins/datasource

I removed that and, inspired by #48, used `grafana-cli` to do the installation. The plugin was installed in the right place:

    /var/lib/grafana/plugins

And it seems to work fine.

Thank you guys for the support. Sorry for sending noise to this thread.

(Also, it looks from @StephenSorriaux's message that this issue could be closed.)
